forte/data/readers/largemovie_reader.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2019 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+A reader to read reviews and IMDb scores from `Large Movie Reviews` dataset for sentiment classification.
+
```

Comment:
If I download the dataset from the link, how can I use this reader? We can write the command in the docstring to help the users.

Reply:
I mean you can make it clear how we can use the reader, like what input parameter should be used.